File: chart/src/main/java/cn/jingzhuan/lib/chart2/event/OnHighlightListener.java
package cn.jingzhuan.lib.chart2.event;

import cn.jingzhuan.lib.chart2.component.Highlight;

/**
 * Created by donglua on 8/23/17.
 */
@FunctionalInterface
public interface OnHighlightListener {

    void highlight(Highlight[] highlights);

}

File: chart/src/main/java/cn/jingzhuan/lib/chart3/event/OnHighlightListener.kt
package cn.jingzhuan.lib.chart3.event

import cn.jingzhuan.lib.chart3.Highlight

/**
 * @since 2023-09-13
 * created by lei
 * 光标变化监听
 */
interface OnHighlightListener {

    fun onHighlightShow(highlight: Highlight?)

    fun onHighlightHide()
}